bundle/tests/scorecard/kuttl/test-node-port/00-install.yaml

```diff
@@ -0,0 +1,16 @@
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: test-node-port
+spec:
+  finalizers:
+  - kubernetes
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: test-node-port-secret
```

Comment:
I don't see this namespace getting used or deleted in 03. Am I missing something? The env doesn't have a namespacedName.